amazon-kinesis-client-multilang/src/main/java/software/amazon/kinesis/multilang/config/KinesisClientLibConfigurator.java

```diff
@@ -62,14 +64,34 @@ public KinesisClientLibConfigurator() {
     public MultiLangDaemonConfiguration getConfiguration(Properties properties) {
         properties.entrySet().forEach(e -> {
             try {
-                utilsBean.setProperty(configuration, (String) e.getKey(), e.getValue());
+                Object value = e.getValue();
+                if(value instanceof String){
+                    value = ((String) value).trim();
```

Comment:
Is there any case where the property value could have leading/trailing spaces?

Reply:
+1, does this break any backwards compatibility where customers could have accidentally included spaces in their property values previously that won't work now?

Reply:
good callout, i don't believe so but ill remove that to be safe.